.travis.yml

```diff
@@ -28,12 +28,11 @@ script:
   )
 - >
   echo "Checking for license headers..." && (
-    FILES=$(grep -rL 'https://www.apache.org/licenses/LICENSE-2.0' cmd internal pkg | grep -vE ".(json|ssa)")
+    FILES=$(grep -rL 'https://www.apache.org/licenses/LICENSE-2.0' cmd internal pkg | grep -vE ".(yaml|ssa)")
```

Comment:
Presumably just swept up in a find-and-replace, but the advantage of comments in YAML is that we can and should include the license header.

Reply:
SG. Done.